modules/tide_cms_support/tide_cms_support.install

```diff
@@ -15,7 +15,7 @@ function tide_cms_support_install() {
   // Create Tide CMS Help block.
   $block = Block::create([
     'id' => 'admin_tide_cms_help',
-    'theme' => \Drupal::config('system.theme')->get('admin'),
+    'theme' => empty(\Drupal::config('system.theme')->get('admin')) ? 'seven' : \Drupal::config('system.theme')->get('admin'),
```

Comment:
@edyuenyw got any idea why `admin` is empty in the reference site?

Reply:
I think, content-reference does not have admin setup. E.g usually after every build, the redirection was to a one time login user. But I'm assuming here.